dev_guide/configmaps.adoc

```diff
@@ -307,27 +285,39 @@ objects in pods.
 [[configmaps-use-case-consuming-in-env-vars]]
 === Consuming in Environment Variables
 
-A `*ConfigMap*` can be used to populate the value of command line arguments. For
-example, consider the following `*ConfigMap*`:
+ConfigMaps can be used to populate individual environment variables or be used in
++its entirety.  As an example, consider the following ConfigMaps:
```

Comment:
"or can populate environment variables from all keys that form valid environment variable names".